templates/workspace_services/guacamole/porter.yaml

```diff
@@ -101,11 +101,16 @@ outputs:
     type: string
     applyTo:
       - install
+  - name: authentication_callback_uri
```

Comment:
It should probably be under uninstall as well, since if we will try to uninstall it will not find the var

Reply:
This is an output param, so it doesn't need to apply to the uninstall too.